### PR TITLE
Made URLs in plugin descriptions clickable.

### PR DESCRIPTION
**Pull request by mattmess1221**
They will not be clickable if the url is invalid.

### CODE REVIEW

File: src/main/java/org/spongepowered/common/command/SpongeCommand.java [hunk 1]
```diff
@@ -476,8 +479,16 @@ private static CommandSpec getPluginsCommand() {
 
                             appendPluginMeta(builder, "ID", container.getId());
                             appendPluginMeta(builder, "Description", container.getDescription());
-                            appendPluginMeta(builder, "URL", container.getUrl());
-
+                            appendPluginMeta(builder, "URL", container.getUrl().map(url -> {
+                                ClickAction.OpenUrl action = null;
+                                try {
+                                    // make the url clickable
+                                    action = TextActions.openUrl(URI.create(url).toURL());
```
Comment (simon816) [inline src/main/java/org/spongepowered/common/command/SpongeCommand.java]: Why not `new URL(url)`?

Comment (mattmess1221) [reply src/main/java/org/spongepowered/common/command/SpongeCommand.java]: I've always just created them from a `URI`. Should I change it to `URL`?

File: src/main/java/org/spongepowered/common/command/SpongeCommand.java [hunk 2]
```diff
@@ -476,8 +479,16 @@ private static CommandSpec getPluginsCommand() {
...
-
+                            appendPluginMeta(builder, "URL", container.getUrl().map(url -> {
+                                ClickAction.OpenUrl action = null;
+                                try {
+                                    // make the url clickable
+                                    action = TextActions.openUrl(new URL(url));
+                                } catch (MalformedURLException e) {
+                                    // or not
+                                }
+                                return Text.builder(url).onClick(action);
```
Comment (simon816) [inline src/main/java/org/spongepowered/common/command/SpongeCommand.java:490]: Could even make it underlined and blue to make it clear it's a link

Comment (kashike) [reply src/main/java/org/spongepowered/common/command/SpongeCommand.java:490]: :-1: for underlining, it makes it hard to read with a lot of resource packs, and partly with vanilla.

**Review by stephan-gh — APPROVED** [review]

**Pull request merged**